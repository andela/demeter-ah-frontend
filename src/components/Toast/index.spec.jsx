import React from 'react';
import { shallow } from 'enzyme';
import { Toast } from './index';

describe('Toast Component', () => {
  it('Should render without errors', () => {
    const component = shallow(<Toast output="welcome" />);
    expect(component).toMatchSnapshot();
  });
});
