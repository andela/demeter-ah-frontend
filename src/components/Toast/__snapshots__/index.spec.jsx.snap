// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toast Component Should render without errors 1`] = `
<h4
  className="text-center p-2"
>
  welcome
</h4>
`;
